pkg/datapath/iptables/iptables_test.go

```diff
@@ -107,6 +107,9 @@ func init() {
 func (s *iptablesTestSuite) TestRenameCustomChain(c *check.C) {
 	mockIp4tables := &mockIptables{c: c, prog: "iptables"}
 	mockIp4tables.expectations = []expectation{
+		{
+			args: "-t mangle -L CILIUM_PRE_mangle",
+		},
```

Comment:
In order to not ignore any meaningful error, when we rename a chain we need to first test if the chain exists (so that we don't return an error in case it doesn't)

Reply:
Oh, I see. Thanks!